### ISSUE TITLE
[🐛 BUG]: protobuf compiler plugin segfaults on import statements

### ISSUE THREAD

**Issue by phroggyy** [roadrunner-server/roadrunner#1337]
### No duplicates 🥲.

- [X] I have searched for a similar issue in our bug tracker and didn't find any solutions.

### What happened?

When compiling protobuf definitions using the `protoc-gen-php-grpc` plugin, the plugin segfaults due to a nil pointer when the service has an `import` statement.

## Steps to reproduce

I have a reproduction repo at https://github.com/phroggyy/protoc-php-failure-sample.

## Expected behaviour

The plugin generates server output even when imports are referenced.

## Additional details

None yet. I plan to dig into some of the plugin's source and will update this ticket if I find further information.

### Version (rr --version)

`protoc-gen-php-grpc 2.11.4` (latest)

### Relevant log output

```shell
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x102d259d0]

goroutine 1 [running]:
github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php.newNamespace(0x14000077d08?, 0x14000186800, 0x1400019c190)
	github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php/ns.go:49 +0x60
github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php.body(0x14000186800?, 0x14000186800, 0x1400019c190)
	github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php/template.go:80 +0x50
github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php.generate(0x14000064180?, 0x1?, 0x1400019c190)
	github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php/generate.go:52 +0x5c
github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php.Generate(0x1400007e070)
	github.com/roadrunner-server/grpc/protoc_plugins/v2/protoc-gen-php-grpc/php/generate.go:38 +0x11c
main.main()
	./main.go:40 +0x40
```
```

**Comment by rustatian** [comment]
Hey @phroggyy 👋🏻 

Yeah, that's because you should add the `package` specifier: https://developers.google.com/protocol-buffers/docs/proto3#packages in your proto.

But either, we should not try to deref a nil pointer for the optional feature :) I'll fix that in a moment.

**Comment by phroggyy** [comment]
We definitely _should_ add a `package` :smile: ain't nothing like a bit of legacy to spice things up though!

**Comment by rustatian** [comment]
Could you please verify, that with `package` it works?

**Comment by phroggyy** [comment]
Confirmed.

**Comment by rustatian** [comment]
done: https://github.com/roadrunner-server/grpc/pull/50

**Comment by rustatian** [comment]
Thank you, @phroggyy 👍🏻 

Merged, will be released on Thursday (3rd of November) (v2.12.0-beta.x). Please keep the `package` statement in your proto until that.
